customer_data_platform/authorized_api/customer_data_platform_log_api.yaml

```
@@ -0,0 +1,157 @@
+openapi: 3.1.0
+info:
+  title: Log API Documentation
+  version: 0.0.0
+  description: This is the documentation of the Log API service.
+servers:
+  - url: https://organization.piwik.pro
+paths:
+  /api/cdp/log/v1/app/{app_id}/audience/{audience_id}/version/{audience_version}/profile/count:
+    get:
+      summary: Count profiles in audience
+      security:
+        - JWTAuth: []
+      operationId: get-log-audience-version-profile-count
+      description: Audience's profiles count
+      parameters:
+        - $ref: "../components.yaml#/components/parameters/AppId"
+        - in: path
+          name: audience_id
+          description: An audience UUID for which profiles count is requested
+          schema:
+            type: string
+            format: uuid
+            example: 81217cfd-c29e-4c7b-9310-936598339971
+          required: true
+        - in: path
+          name: audience_version
+          description: An audience version for which we are counting profiles
+          schema:
+            type: integer
+            minimum: 0
+            example: 2
+          required: true
+      responses:
+        200:
+          content:
+            application/vnd.api+json:
+              schema:
+                type: integer
+                minimum: 0
+                example: 14
+          description: Number of profiles in given audience version
+        400:
+          $ref: "../components.yaml#/components/responses/BadRequestError"
+        401:
+          $ref: "../components.yaml#/components/responses/UnauthorizedError"
+        402:
+          $ref: "../components.yaml#/components/responses/ModuleDisabledError"
+        404:
+          $ref: "../components.yaml#/components/responses/NotFoundError"
+        422:
+          $ref: "../components.yaml#/components/responses/UnprocessableEntityError"
+        500:
+          $ref: "../components.yaml#/components/responses/InternalServerError"
+  /api/cdp/log/v1/app/{app_id}/audience/{audience_id}/version/{audience_version}/profile/last:
+    get:
+      summary: List last profiles belonging to given audience
+      security:
+        - JWTAuth: []
+      operationId: get-log-audience-version-profile-last
+      description: Returns at most 1000 items
```

While technically true, this sentence might suggest that you can't get more than 1k profiles. Let's clarify this.
```suggestion
      description: By default, returns the last 100 items. With the use of limit and offest parameters, it is possible to extract all profiles belonging to the audience.
```